public/manifest.json

```diff
@@ -7,6 +7,7 @@
   "name": "ShapeShift",
   "short_name": "ShapeShift",
   "description": "Your Web3 & DeFi Portal",
+  "iconPath": "/icon-512x512.png",
```

Comment:
Preparation for ShapeShift as a SAFE app integration, since EIP-6963 means we will now be able to work as an app within SAFE

Reply:
https://help.safe.global/en/articles/145503-how-to-create-a-safe-app-with-safe-apps-sdk-and-list-it